main.js

```diff
@@ -49,9 +49,9 @@ function createMenu() {
   let template = [
     {
       id: 'updateCheck',
-      label: pendingVersionUpdate ? ('Download Workstation v' + pendingVersionUpdate)  : 'Check for updates...',
-      enabled: workstation.areUpdatesEnabled,
-      click: () => { triggerUpdateCheck(true) }
+      label: pendingUpdate ? ('Download update v' + pendingUpdate.version)  : 'Check for updates...',
```

Comment:
Is there a reason we want the version number in the menu item?
We should also be using title case for menu items ... so "Download Update"

Reply:
We can remove the version and just go with: 'Download Update...'